layouts/partials/meta_json_ld.html

```diff
@@ -3,6 +3,7 @@
     "@context": "https://schema.org",
     {{ if .IsPage -}}
     "@type": {{ if or (eq .Section "blog") (eq .Section "post") }}"BlogPosting"{{ else }}"WebPage"{{ end }},
+    {{ if not (or (eq .Section "blog") (eq .Section "post")) }}"breadcrumb": {{ partial "meta_breadcrumb_json_ld.html" . }},{{ end }}
```

Comment:
Why do we not want bredcrumbs on blog and post pages?

Reply:
According to `schema.org`:

Values expected to be one of these types
[BreadcrumbList](https://schema.org/BreadcrumbList)
[Text](https://schema.org/Text)
Used on these types
[WebPage](https://schema.org/WebPage)

Unless I've misunderstood (quite possible), that means validation would not accept a breadcrumb in a `BlogPost`.

Reply:
Which reminds me, this should a configurable set of sections... I had planned to do that already.

Reply:
I suspect schema.org's reasoning is that a blog post is typical only a single level, so doesn't need a breadcrumb; but I am guessing there.

Reply:
> Which reminds me, this should a configurable set of sections... I had planned to do that already.

I think it would be best to have a opt in setting, breadcrumbs are only built for the listed section. If not set the code does not run.

That would also handle the Hugo v0.109.0 requirement. We can add docs that spells this out. "Do not set this setting unless you run v0.109.0."

Style the setting on `mainSections`. Maybe name it something like:

* `breadcrumbSections`
* `metaBreadcrumbSections`
* `schemaBreadcrumbSections`

Reply:
Added a `blogSections` (optional; default is 'post' and 'blog' to behave as existing logic) site parameter and `breadcrumbSections` parameter (optional: if it is present and the sections are not in in `blogSections`, then the breadcrumbs are created).

Also added a brief documentation in the README.